google/generativeai/client.py

```diff
@@ -119,7 +116,6 @@ def configure(
         api_key: str | None = None,
         credentials: ga_credentials.Credentials | dict | None = None,
         # The user can pass a string to choose `rest` or `grpc` or 'grpc_asyncio'.
-        # See `_transport_registry` in `DiscussServiceClientMeta`.
         # Since the transport classes align with the client classes it wouldn't make
```

Comment:
```suggestion
        # See _transport_registry in the google.ai.generativelanguage package.
        # Since the transport classes align with the client classes it wouldn't make
```